Library/Homebrew/cask/artifact/abstract_uninstall.rb

```diff
@@ -385,21 +385,29 @@ def trash_paths(*paths, command: nil, **_)
         [trashed, untrashable]
       end
 
-      def uninstall_rmdir(*directories, command: nil, **_)
-        return if directories.empty?
+      def recursive_rmdir(*directories, command: nil, **_)
+        return true if directories.empty?
 
-        ohai "Removing directories if empty:"
         each_resolved_path(:rmdir, directories) do |path, resolved_paths|
```

Comment:
This should also use `.all?` now.

Reply:
@reitermarkus Sorry, I don't follow… Does `all?` not need to operate on an array of _already_ `resolved_paths` (therefore inside this method call)? Also, the second `directories` argument can be a [glob pattern] string…